### DISCUSSION TITLE
能否提供3.2.7.3对应的分支源码

### DISCUSSION THREAD

**Discussion by hergua** [TencentBlueKing/bk-job#492] [General]
因定制需要需要更改蓝鲸【社区版】6.0.4 中作业平台的显示，能否给出3.2.7.3 RELEASE分支相关的源码呢

**Comment by wangyu096** [comment]
你好，具体版本可以从tag里边checkout。
不过蓝鲸作业平台开源版本最低的是从3.3.4.1开始的，之前的tag在这个仓库已经全部被清理了。如果要定制的话，社区版6.0 job对应的最新的稳定版本在3.3.x分支，建议用这个分支

**Reply by hergua** [reply]
关于蓝鲸社区版，升级job版本的操作，有相关的操作手册或说明吗

**Reply by wangyu096** [reply]
https://bk.tencent.com/

可以去蓝鲸社区，有相关的支持
